cmd/earthly/main.go

```diff
@@ -3205,6 +3329,257 @@ func (app *earthlyApp) actionListTargets(c *cli.Context) error {
 	return nil
 }
 
+func (app *earthlyApp) useSatellite(c *cli.Context, satelliteName, orgName string) error {
+	inConfig, err := config.ReadConfigFile(app.configPath)
+	if err != nil {
+		if c.IsSet("config") || !errors.Is(err, os.ErrNotExist) {
+			return errors.Wrap(err, "read config")
+		}
+	}
+
+	newConfig, err := config.Upsert(inConfig, "satellite.name", satelliteName)
+	if err != nil {
+		return errors.Wrap(err, "could not update satellite name")
+	}
+	// Update in-place so we can use it later, assuming the config change was successful.
+	app.cfg.Satellite.Name = satelliteName
+
+	newConfig, err = config.Upsert(newConfig, "satellite.org", orgName)
+	if err != nil {
+		return errors.Wrap(err, "could not update satellite name")
+	}
+	app.cfg.Satellite.Org = orgName
+	err = config.WriteConfigFile(app.configPath, newConfig)
+	if err != nil {
+		return errors.Wrap(err, "could not save config")
+	}
+
+	return nil
+}
+
+func (app *earthlyApp) printSatellites(satellites []cloud.SatelliteInstance) {
+	for _, satellite := range satellites {
+		if app.cfg.Satellite.Name == satellite.Name {
+			app.console.Printf("* %s", satellite.Name)
+		} else {
+			app.console.Printf("  %s", satellite.Name)
```

Comment:
Nit: Ideally, we should use `stdout` for these (`app.console` uses `stderr`, I believe).

Reply:
Will do. Didn't realize `app.console.Printf` was also going to stderr.